# AID.cry-UPDSeq
**Functions/scripts used in the AID.cry UPD-Seq publication**

Step1: Do the Sequence alignment and create .bam files    
Step2: use GCmmDepth to extract depth of coverage after applying the GCmm Filter    
Step3: Calculate Differential Coverage using the NDC2 fuction       
Step4: Find peaks using a threshold (5x the standard deviation of NDC output)       
Step5: Extract nucleotide counts as described here: https://github.com/rayanramin/U-idx#readme    
Step6: Calculate UI using the UX.R function and perform the Analysis (follow UI/UI Analysis)

######################################################

If you need help or have any questions regarding the code,     
please contact the corresponding author or Ramin Sakhtemani:       
rsakhtemani _(at)_ mgh.harvard.edu

#####################################################
